examples/tweet_generator.py

```diff
@@ -0,0 +1,155 @@
+import io
+import re
+import textwrap
+import urllib.request
+from datetime import datetime
+from PIL import Image, ImageFont, ImageDraw, ImageOps
+from pincer import command, Client, Descripted
+from pincer.objects import Message, Embed, MessageContext
+
+
+class Bot(Client):
+
+    @Client.event
+    async def on_ready(self):
+        print(
+            f"Started client on {self.bot}\n"
+            "Registered commands: " + ", ".join(self.chat_commands)
+        )
+
+    @command(
+        name='twitter',
+        description='to create fake tweets',
+        guild=690604075775164437
```

Comment:
I think this shouldn't be in the example

Reply:
you're right, I forgot it there